zebra-chain/src/sapling/shielded_data.rs

```diff
@@ -27,12 +50,12 @@ pub struct ShieldedData {
     /// methods provide iterators over all of the [`Spend`]s and
     /// [`Output`]s.
     #[serde(with = "serde_helpers::Either")]
-    pub first: Either<Spend, Output>,
+    pub first: Either<Spend<PerSpendAnchor>, Output>,
```

Comment:
Hard-coded `PerSpendAnchor` will break when we implement V5:
```suggestion
    pub first: Either<Spend<AnchorV::PerSpend>, Output>,
```